### ISSUE TITLE
Pip install warns about externally managed environment

### ISSUE THREAD

**Issue by revevil** [alphacep/vosk-api#1618]
Hello,

I have used vosk with nerd-dictation on the last version of Linux Mint v21.3, but vosk is broken on the new Linux Mint v22. I use the command "pip3 install vosk". That used to work beautiful, but now I get this error message:

error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.

My pip version is
pip 24.0 from /usr/lib/python3/dist-packages/pip (python 3.12)
but I checked my python3 version with: python3 --version and I received this output:
Python 3.12.3

So I know my python3 is out of date because vosk specifies that python3 should be version 3.5-3.9. That might be the whole problem right there, but I'm not sure.

I also tried pip3 install python3-vosk but received the same error message. I am using Linux Mint MATE v22. I just wanted to ask you for help, because this program worked excellent before. If you need any more information from me please let me know.

Thanks,

-revevil

**Comment by nshmyrev** [comment]
You can check

https://stackoverflow.com/questions/75602063/pip-install-r-requirements-txt-is-failing-this-environment-is-externally-mana

**Comment by revevil** [comment]
> You can check
> 
> https://stackoverflow.com/questions/75602063/pip-install-r-requirements-txt-is-failing-this-environment-is-externally-mana

Thank you very much! I am studying this page, and I will report back as soon as I decide which solution to try. Some of the solutions are a little scary, so I will research them a bit more.

**Comment by revevil** [comment]
I have chosen a solution, and this one seems to work quite well. Again THANK YOU so much for pointing me in the direction of that discussion. They provided multiple solutions to the problem. The one I have chosen is this:

sudo rm /usr/lib/python3.*/EXTERNALLY-MANAGED

After installing pip with "sudo apt install python3-pip" I was able to install vosk with "pip3 install vosk" just fine. After installing vosk, I installed xdotool with "sudo apt install xdotool" and then proceeded to install nerd-dictation via the instructions provided on the github page. Everything is working 100% great! Thanks again! This issue is completely resolved.
